src/boxes/wide.rs

```diff
@@ -0,0 +1,50 @@
+/*
+ * © 2022 Samir Dharar
+ * All rights reserved.
+ */
+
+use crate::boxes::Mp4Box;
+use crate::Error;
+use crate::Header;
+use std::io::{Read, Seek};
+
+#[derive(Debug)]
+pub struct Wide {
```

Comment:
If this struct doesn't its own methods and fields, why not handle it with `InnerAtom`?

Reply:
I'm following your base logic/structure here, as none of the first level atoms like ftyp, free, mdat have their own methods and fields but are under separate files. Do they also require handling with InnerAtom to avoid redundancy?

Reply:
I've done this in the newer commit on the stco-parsing branch.